### PR TITLE
Prepare sample module gradle file for being converted to Kotlin DSL

### PR DESCRIPTION
**Pull request by oas004**
#### WHAT
Prepare the sample module' gradle file to be converted easily to Kotlin DSL

#### WHY
Following up #846  relating to #833 

#### HOW
Following https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html#prepare_your_groovy_scripts

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: sample/build.gradle [hunk 1]
```diff
@@ -14,49 +14,49 @@
  * limitations under the License.
  */
 
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
+apply plugin: "com.android.application"
+apply plugin: "kotlin-android"
 apply plugin: "com.google.protobuf"
 apply plugin: "com.google.android.libraries.mapsplatform.secrets-gradle-plugin"
```
Comment (oas004) [inline sample/build.gradle:20]: Should I maybe also migrate this to use the plugin lambda?

Comment (luizgrp) [reply sample/build.gradle:20]: I think so, as it's strongly recommended in that doc!

Comment (oas004) [reply sample/build.gradle:20]: Okay, I'll fix it :)

**Review by luizgrp — APPROVED** [review]
LGTM

**Pull request merged**